cdap-app-fabric/src/main/java/io/cdap/cdap/internal/bootstrap/guice/BootstrapModules.java

```diff
@@ -72,6 +72,21 @@ protected void configure() {
     };
   }
 
+  /**
+   * @return bootstrap module that do not need to execute bootstrap steps.
```

Comment:
fix checkstyle issues in the files changed.

Reply:
Done.